_includes/tutorials/generate-test-data-streams/kafka/markup/dev/consume-topic-02b.adoc

```diff
@@ -0,0 +1,10 @@
+We now have two Kafka topics being written to. The first (`devices`) is keyed on the MAC address, as can be seen from the data: 
```

Comment:
This file has duplicate content to `consume-topic-02a.adoc`.

Reply:
Good catch, thanks @bbejeck. Fixed.